manifests/concourse-manifest/concourse-base.yml

```diff
@@ -99,6 +102,7 @@ jobs:
         forward_host: 127.0.0.1
         atc:
           address: 127.0.0.1:8080
+          old_resource_grace_period: 180m
```

Comment:
Initially, concourse announced default caching period was 24 hours. Then they realized they set it to 5 mins instead. Hence I'd say both values are (and anything in between is) safe. As this is caching only, it should not affect anything. Resource `check` methods should properly return new versions, which would invalidate old cache. I set this to 3 hours so it can last for 2 pipeline passes (as currently, a single pass including acceptance tests can take ~1h10-20 mins). I can also imagine that caching uses some disk space. I think if we are running out of it, we should increase it instead of decreasing cache period (or disabling caching).